wallet/src/de/schildbach/wallet/ui/invite/InviteHandler.kt

```diff
@@ -185,21 +214,21 @@ class InviteHandler(val activity: AppCompatActivity, private val analytics: Anal
                 (errorMessage.contains("IdentityAssetLockTransactionOutPointAlreadyExistsError")) -> {
                     showInviteAlreadyClaimedDialog(blockchainIdentityData.invite!!)
                     // now erase the blockchain data
-                    getInstance().clearBlockchainData()
+                    getInstance().clearBlockchainIdentityData()
```

Comment:
The method was renamed because it does not clear the blockchain, but only Blockchain Identity Data.